src/test/code-sample.test.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from '@esm-bundle/chai';
+import '../lib/code-sample.js';
+import {CodeSampleElement} from '../lib/code-sample.js';
+import {html, render} from 'lit-html';
+import {UpdatingElement} from 'lit-element';
+import {CodeMirrorEditorElement} from '../lib/codemirror-editor.js';
+import {CodeSampleProjectElement} from '../lib/code-sample-project.js';
+
+describe('code-sample', () => {
```

Comment:
Most other places we're using ttd-style test/suite

Reply:
Done.